server/terraform/modules/myhealth/main.tf

```diff
@@ -257,3 +259,62 @@ resource "google_compute_backend_service" "backend" {
   health_checks = null
   depends_on    = [google_project.project]
 }
+
+
+# Google Cloud bucket for static content assets.
+resource "google_storage_bucket" "content" {
+  name     = "${var.project_id}-static-content" #################### Should a prefix be added to make uniqueness more likley? 
```

Comment:
There's already uniqueness in project ids.... so I'd think it unlikely that someone would coincidentally use the same prefix. There's been no collisions so far... so we'll just worry about it if and when it happens.

It seems like the names have to be globally unique:https://cloud.google.com/apis/design/resource_names

Reply:
Right, there's multiple namespaces.
$project_id = globally unique. Flat namespace. (Sort of; there are or were a certain class of scoped IDs but I think that's gone?) Not reusable. One of two true immutable indentifiers for the Google Cloud project. Exposed in a few places, including the appengine URL (foo.appspot.com, though for reasons I don't understand they added foo.someregionalgibberish.appspot.com) .
Various resources which are scoped to the project itself, e.g. most (but not all) of the things using projectid.something in this config.
Bucket ID. Globally unique. Both flat and domain owned namespace. Names can be re-used shortly after they are purged. (Note: every project gets a bucket named $projectid.appspot.com, which is guaranteed to be available due to the properties of the projectid described above).

Anyway, yes using $projectid-static-content has a very high chance of being available; if it ever breaks this config will just need more wackyness to manage it.

In any case, the app code needs to be updated to use this new bucket URL, be it using this name or the load balancer which hides the bucket name.